README.md

```diff
@@ -326,7 +326,7 @@ This repository also provides one such platforms where contributers come over an
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
 |[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
-
+[GoFish](https://github.com/kunjgit/GameZone/tree/main/Games/GoFish)
```

Comment:
there should be bars as other games had in the ending

Reply:
@ishita-43  Made the changes